spyder/plugins/statusbar/plugin.py

```diff
@@ -53,11 +53,13 @@ class StatusBar(SpyderPluginV2):
     def get_name():
         return _('Status bar')
 
-    def get_icon(self):
-        return self.create_icon('statusbar')
+    @classmethod
+    def get_icon(cls):
+        return cls.create_icon('statusbar')
 
-    def get_description(self):
-        return _('Provide Core user interface management')
+    @staticmethod
+    def get_description():
+        return _("Manage main window status bar.")
```

Comment:
```suggestion
        return _("Display the main window status bar.")
```

This doesn't just allow managing the status bar (ala some sort of preferences), but also displaying the items there in the first place. Also avoids overuse of "manage"